storio-content-resolver/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
 
     provided rootProject.ext.supportAnnotations
     provided rootProject.ext.rxJava
+    provided rootProject.ext.rxAndroid
```

Comment:
Or, if you really need this for tests (but I don't see reasons for that) -> use `androidTestProvided` configuration